deps/graph-parser/src/logseq/graph_parser/db.cljs

```diff
@@ -0,0 +1,15 @@
+(ns logseq.graph-parser.db
```

Comment:
I think it would be helpful soon to have a separate db library that houses reusable db fns. It would be useful to move these `logseq.graph-parser.db.*` namespaces as well as the rules one as I have had to copy it in a couple projects. A separate db library could also be a handy way to have a db protocol if we want to try out a datalog alternative like https://github.com/clj-easy/tools.misc does with json libs. Would you be ok with this @tiensonqin?

Reply:
@tiensonqin Would it be ok if I made a `deps/db` for the above? My plan would be move `logseq.graph-parser.db.*` namespaces and `frontend.db.rules` under `logseq.db`. My motivation is to have common rules readily available to nbb scripts. Alternatively, I could just move `frontend.db.rules` into `deps/graph-parser` if you don't want a separate db dependency yet

Reply:
Sure, moving under `logseq.db.*` and making a `deps/db` will be nice.